sdk/apicenter/Azure.ResourceManager.ApiCenter/CHANGELOG.md

```diff
@@ -1,17 +1,11 @@
 # Release History
 
-## 1.0.0-beta.3 (Unreleased)
+## 1.0.0-beta.3 (2024-04-09)
```

Comment:
Can this be stable as the swager spec is stable version.

Reply:
@azaslonov thoughts?

Reply:
Yes please, it should be stable 1.0.0 version. @alexyaang could you please change?

Reply:
changed it to `1.0.0`, good catch guys